src/xebia_email_signature/static/offices.json

```diff
@@ -1,11 +1,11 @@
 {
   "Hilversum, Netherlands": {
     "address": "Laapersveld 27, 1213 VB Hilversum",
-    "telephone": "+31355381921"
+    "telephone": "+310355381921"
```

Comment:
This is incorrect and will not connect

Reply:
Thanks, i see that fix for that was released two days ago and because we worked with bitbucket and than had to come back here, i did not have it synced. I will be doing some quick fixes right now so i will add it to next PR

Reply:
It's not correct and has impact on the current signature generator. But the new one isn't using the JSON at all?

Reply:
It is not, as in new design we dont have this dropdown to select an office